public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -540,11 +542,17 @@ export class ElasticDatasource extends DataSourceApi<ElasticsearchQuery, Elastic
   }
 
   getMultiSearchUrl() {
+    const searchParams = new URLSearchParams();
+
     if (this.esVersion >= 70 && this.maxConcurrentShardRequests) {
-      return `_msearch?max_concurrent_shard_requests=${this.maxConcurrentShardRequests}`;
+      searchParams.append('max_concurrent_shard_requests', '' + this.maxConcurrentShardRequests);
+    }
+
+    if (this.esVersion >= 70 && this.includeFrozen) {
+      searchParams.append('ignore_throttled', 'false');
     }
 
-    return '_msearch';
+    return (`_msearch?` + searchParams.toString()).replace(/\?+$/, '');
```

Comment:
So this is where the non ideal situation of having both frontend and backend implementation comes into play. There is basically the same code here: https://github.com/grafana/grafana/blob/5070f7a75bba31f831000c37203b6f8668d15bf5/pkg/tsdb/elasticsearch/client/client.go#L313

So this has to be also implemented there so that alerting requests work the same way (although hard to say if alerting over frozen indices makes much sense).